packages/modeling/src/curves/bezier/tangentAt.js

```diff
@@ -1,7 +1,7 @@
 /**
  * Calculates the tangent at a specific point along a bezier easing curve.
  * For multidimensional curves, the tangent is the slope of each dimension at that point.
- * See the bezier_extrudeAlongPath.js example to see this in use.
+ * See the extrudeAlongPath.js example to see this in use.
```

Comment:
thought...

if we give the examples JSDOC strings and name properly then the code can link to example documentation, as well as the opposite.

Reply:
Yeah, not a bad idea.